README.md

```diff
@@ -446,6 +454,28 @@ Found '/path/to/project/.nvmrc' with version <5.9>
 Now using node v5.9.1 (npm v3.7.3)
 ```
 
+`.nvmrc` files may have comments. The following is the notation to have a comment inside a `.nvmrc` file:
+```bash
+# This is a comment
+
+  node # Another comment
+```
+When reading the `.nvmrc` file, nvm will:
+1. Remove all comments on their own line as shown in the first line of the example
+2. Remove all comments in middle of a line. Because of this, # is not allowed inside your `nvm` alias. Example on line 3
+3. Remove all newlines such as the one on line 2
+4. Remove all tabs, such as the one on line 3
+5. Remove all spaces, such as the ones between `node` and `# This is another comment`, because of this, `nvm` aliases may not have a space
+
+For a preview of how your file looks without comments, check out the `nvm cat` command:
+
+##### nvm cat
+The following is the usage for `nvm cat`:
+```
+nvm cat [-pc] [--silent] -f [file] [header]
+```
+The `--path` or the `-p` boolean flag indicates whether the file provided is the absolute path to the file. If it is not true, then `nvm` will search for the file provided. The `-c` or the `--comments` boolean flag determines whether comments will be stripped or not by `nvm`. The `--silent` flag enables silent mode. The `-f` option allows you to provide the file that is being read by `nvm`. The default option for `-f` is `.nvmrc`. An alias for `-f` is `--file`. Any other argument that matches the wildcard * will be appended to the header. The header is printed out before the file's contents are. If the header is not provided then the `NVM_CAT_HEADER` enviormental variable. This variable is not set by default. You can set it to change your header. Any instance of `:FILE:` will be replaced by the file being read in the header.
```

Comment:
adding an entire top-level command just to see what version `nvm use` would end up using seems like a lot of complexity for virtually no gain.